iphone/Classes/TiPlatformDisplayCaps.m

```diff
@@ -30,17 +30,19 @@ - (NSString *)apiName
   return @"Ti.Platform.DisplayCaps";
 }
 
-- (id)dpi
+- (NSNumber *)dpi
 {
   return [NSNumber numberWithInt:[TiUtils dpi]];
 }
 
+// TODO Remove? Not in our docs!
```

Comment:
Yep. Android also does not support it. Probably a left-over? We could git-blame to see the details if required.